README.md

```diff
@@ -1,6 +1,13 @@
 # Kottans-frontend
 # Help with your first commit -> [discord](https://discord.gg/k5zS4XmH)
 ___
+## **Hi there!**
+Despite I've got only paws I'm looking forward to learn and try something new and interesting!
+
+<img src="https://github.com/lometari/mock-repo/raw/firstStep/gif/belovedCat.jpg" width="70%" margin="auto">
```

Comment:
no HTML tegs in markdown pls 
you can read more about hot to do it here -> https://www.markdownguide.org/basic-syntax/